src/routes/posts.js

```diff
@@ -78,9 +78,37 @@ const deletePost = async (req, res, next) => {
     }
 }
 
+// UPDATE AN ARTICLE
+
+const updatePost = async (req, res, next) => {
+    const { id } = req.params // WHERE
+    const { title, content, image, published } = req.body // SET
+    try {
+        const updateArticle = await db.query(
```

Comment:
`updateArticle` denotes an action and is suitable if it was a function name but not too readable here. Kindly update to something a bit more readable. even `result` or something better might be nice

Reply:
i will stick with `result`